addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -243,7 +244,21 @@ export class PivotCorePlugin extends OdooCorePlugin {
             }
         }
 
-        this._addBorders(sheetId, position, table);
+        const pivotZone = {
+            top: position.row,
+            bottom: position.row + pivotCells[0].length - 1,
+            left: position.col,
+            right: position.col + pivotCells.length - 1,
+        };
+        const numberOfHeaders = table.getNumberOfHeaderRows() - 1; // -1 to not include measures in the table headers
```

Comment:
`const numberOfHeaders = table.getColHeaders().length;`

Reply:
`table.getColHeaders().length`and `table.getNumberOfHeaderRows()` give the exact same result, I'm not sure what's teh difference ?